virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -104,33 +103,34 @@ The SVVP Certification applies to:
 * Quick start tours are available for several {VirtProductName} features. To view the tours, click the *Help* icon *?* in the menu bar on the header of the {VirtProductName} console and then select *Quick Starts*. You can filter the available tours by entering the `virtualization` keyword in the *Filter* field.
 
 
-[id="virt-4-14-installation-new"]
-=== Installation
+//[id="virt-4-14-installation-new"]
+//=== Installation
 
 
 [id="virt-4-14-networking-new"]
 === Networking
 //CNV-18090 OVN-Kubernetes secondary network
 * You can connect a virtual machine (VM) to an xref:../../virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc#virt-connecting-vm-to-ovn-secondary-network[OVN-Kubernetes secondary network] by using the web console or the CLI.
 
-[id="virt-4-14-storage-new"]
-=== Storage
+//[id="virt-4-14-storage-new"]
+//=== Storage
 
 
 [id="virt-4-14-web-new"]
 === Web console
 
 * Cluster administrators can now enable automatic subscription for {op-system-base-full} virtual machines in the {VirtProductName} xref:../../virt/getting_started/virt-web-console-overview.adoc#overview-settings-cluster_virt-web-console-overview[web console].
 
-// CNV-29848: Release: Mechanism to manage DP/TP features in the UI
-* Cluster administrators can enable or disable link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] features on the link:https://docs.openshift.com/container-platform/4.14/virt/getting_started/virt-web-console-overview.html#overview-settings-cluster_virt-web-console-overview[*Settings*] tab on the *Virtualization* -> *Overview* page.
+//CNV-18298 force stop
+* You can now force stop an unresponsive VM from the xref:../../virt/getting_started/virt-web-console-overview.adoc#virtualmachine-details-page_virt-web-console-overview[action menu]. To force stop a VM, select *Stop* and then *Force stop* from the action menu.
 
 // CNV-28720
 // pending merge of bootable volumes PR
 * The *DataSources* and the *Bootable volumes* pages have been merged into the xref:../../virt/getting_started/virt-web-console-overview.adoc#bootablevolumes-page_virt-web-console-overview[*Bootable volumes* page] so that you can manage these similar resources in a single location.
 
-//CNV-18298 force stop
-* You can now force stop an unresponsive VM from the xref:../../virt/getting_started/virt-web-console-overview.adoc#virtualmachine-details-page_virt-web-console-overview[action menu]. To force stop a VM, select *Stop* and then *Force stop* from the action menu.
+// CNV-29848: Release: Mechanism to manage DP/TP features in the UI
+* Cluster administrators can enable or disable link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] features on the link:https://docs.openshift.com/container-platform/4.14/virt/getting_started/virt-web-console-overview.html#overview-settings-cluster_virt-web-console-overview[*Settings*] tab on the *Virtualization* -> *Overview* page.
```

Comment:
This needs to be an xref, and I think it should point to the Settings tab (the current link actually points to the Cluster tab, which is in the Settings tab section). IIUC this should be the xref:

```
xref:../../virt/getting_started/virt-web-console-overview.adoc#overview-settings_virt-web-console-overview[*Settings*]
```

Reply:
Done